### ISSUE TITLE
Users appear away even if the user is logged in to the computer

### ISSUE THREAD

**Issue by prakashprabhakar** [mattermost/desktop#238]
I confirm (by marking "x" in the [ ] below): 

- [x] This is not a feature idea. Feature ideas go here: http://www.mattermost.org/feature-ideas/
- [x] This is not a troubleshooting question. Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/
- [x] This is a product defect issue that can be reproduced using the steps listed below.
- [x] I have read [CONTRIBUTING.md](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---

#### Summary

Users appear away even if the user is logged in to the computer

#### Steps to reproduce

- Ubuntu 14.04
- Mattermost Desktop App version: 1.3.0
- Mattermost Server version: 3.2

Steps:

1) Have 2 users logged in.
2) Have User-1 to have logged into mattermost and minimised. He should be working on other things.
3) Have User-2 to check his status.

#### Expected behavior

1) If the user is working on his computer, and mattermost is minimised, then the user should be shown as available (GREEN TICK MARK).

2) If the user is away from the computer for a set period of time, then he should be shown as away (GREEN HYPHEN MARK).

#### Observed behavior

Even though the user is very much working on his computer the user is shown as away just because he has minimised the application.

What did you see happen? Please include relevant error messages and/or screenshots.

**Comment by jasonblais** [comment]
Thanks @prakashprabhakar for the report!

This is currently by design -- if the user is away from the Mattermost app (whether in the browser or desktop), they will be marked as 'away' regardless if they are logged in to their computer.

If you'd like to propose this as a feature request instead, I encourage you to add it to the [Feature Idea forums](https://mattermost.uservoice.com/forums/306457-general) so the proposal can be discussed by the community first. 

Everyone gets 10 votes for features, each influencing the direction of the Mattermost app.

**Comment by prakashprabhakar** [comment]
@jasonblais I am sorry. I had forgotten to reply. Well I would disagree with you on this. Like I remember this is not the case in google chat or slack. The user is shown away only if there is no activity on the machine and not the app. Kindly review. Actually users are getting confused with the status.

**Comment by jasonblais** [comment]
Thanks for the feedback @prakashprabhakar, and sorry for the late response.

We'll consider it in an upcoming UX meeting this week as it's something that affects both the server and the desktop app. I'll come back with an update afterwards.

**Comment by jasonblais** [comment]
Hey @prakashprabhakar, I've made a forum post to hear more feedback from the community. Feel free to chime in?

If we changed the behaviour, it would change when/how email and push notifications work, so we want to think about the change carefully.

**Comment by jasonblais** [comment]
Closing the issue on the desktop repo, we can continue conversation in the forum: https://forum.mattermost.org/t/marking-users-away-from-mattermost-vs-away-from-computer/2173

There is already one forum response, and we're hoping to hear more feedback on the topic. 

One thing to note is that there isn't a way to differentiate who is away from Mattermost vs. from their computer on the web app. So if we carried out with the change suggested here, we would have
 - web app: User set `away` after being away from Mattermost for 5+ minutes
 - desktop: User set `away` after being away from computer for 5+ minutes

which would make the experience inconsistent.
